cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/StreamSession.java

```diff
@@ -180,6 +180,16 @@ List<RingInstance> getReplicas()
         return replicasForTokenRange;
     }
 
+    /**
```

Comment:
Can we have proper java docs so the javadoc task doesn't complain about missing `@param` and `@return` tags?